### PR TITLE
Ru

### PR DESCRIPTION
**Pull request by Aparus**
- [x] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you explained what your changes do, and why they add value to the Guides?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----

### CODE REVIEW

**Comment by Aparus** [comment]
I have translated
* Getting Paid 
* Legal 
(in addition to Starting a Project, that I translated before)

**Comment by nholden** [comment]
👋 Hi, @Aparus! Thanks for this contribution! (And sorry for briefly closing the PR accidentally.)

It looks like many of these files are still in English. Please let us know when the PR is ready for review.

**Comment by Aparus** [comment]
>  Hi, @Aparus! Thanks for this contribution! (And sorry for briefly closing the PR accidentally.)
> 
> It looks like many of these files are still in English. Please let us know when the PR is ready for review.

Yes. I'm sorry for that. I did that for pass tests. But there is still one broken link (in English, Upstream). I don't know how to fix it?

**Comment by itsbagpack** [comment]
@Aparus thank you for pointing that out. It does look like there's a URL (in `main`) that is 404'ing and causing the build to fail, which we'll look into. In the meantime, could you translate the other articles? It also looks like https://github.com/github/opensource.guide/pull/2395 is providing an (incomplete) Russian translation. Perhaps you can work together on a full translation? Just a suggestion.

**Comment by Aparus** [comment]
> @Aparus thank you for pointing that out. It does look like there's a URL (in `main`) that is 404'ing and causing the build to fail, which we'll look into. In the meantime, could you translate the other articles? It also looks like #2395 is providing an (incomplete) Russian translation. Perhaps you can work together on a full translation? Just a suggestion.

Thank you for noticing my efforts. I wish to translate them all, but there is so huge amount of text, like months of work :smile: . I will  try. I also wrote to @alroniks, who also started translation, but he didn't answer yet.

**Comment by moomoo56** [comment]
> _articles / ru / best-practices.md

He

**Review by moomoo56 — APPROVED** [review]

**Review by moomoo56 — APPROVED** [review]

**Comment by Aparus** [comment]
Does anyone know why TOC doesn't work in any language other than English? I see some translators ignore this part of articles, but others translate it, but it doesn't appear anyway...
Screenshot from 2021-09-03 06-10-43
Screenshot from 2021-09-03 06-10-24

**Review by skrsmall — APPROVED** [review]
Vuvuvuvuvu

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]
Need to fix CI failures: https://github.com/github/opensource.guide/pull/2513/checks?check_run_id=3532344454#step:7:38

**Comment by geroi** [comment]
@Aparus Such a great amount of work! I just got 2 pages translated:
- Metrics
- Leadership and Governance
How can we merge our efforts?

**Comment by DNGSlug** [comment]
guys i need guidancei am stuck in Pak and our whole Family's phones have been hacked  

they can even access our cameras we are house bound as it all started when we started transferring money through apps when we came pak and these greedy filths somehow first hacked a wife dongle then my nephew's Huwai(no wonder)and so far 3 iPhones 2 Mac book pros   
my Screen Time passcode jammed/forgot and my apple add is grey as someone on his ipad using it too so i cant sign out as i have forgot the password
it's life threatening kinda situationanyone putting their anger aside and willing to help?!?
regardsNoveed 

Sent from Yahoo Mail for iPhone


On Friday, September 10, 2021, 5:00 am, Alexander Kashmatov ***@***.***> wrote:




@Aparus Such a great amount of work! I just got 2 pages translated:
   
   - Metrics
   - Leadership and Governance   
How can we merge our efforts?

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by Aparus** [comment]
@nholden :
> It looks like many of these files are still in English. Please let us know when the PR is ready for review.
Hi, Nick. I have completed the translation, we worked together with @geroi 
All tests are passed. 
You can merge PR.

File: _articles/ru/leadership-and-governance.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+Ваш проект растет, люди вовлечены в него, и вы полны решимости поддерживать его. На этом этапе вы, возможно, задаетесь вопросом, как включить постоянных участников проекта в свой рабочий процесс, будь то предоставление кому-то доступа к правкам кода (commit) или модерацию дебатов в сообществе. Если у вас есть вопросы, у нас есть ответы.
+
+## Какие примеры формальных ролей используются в проектах с открытым исходным кодом?
+
+Многие проекты имеют схожую структуру распределения ролей и признания заслуг участников.
+
+Однако что на самом деле означают эти роли, зависит только от вас. Вот несколько типов ролей, которые вы можете узнать:
+
+- **Сопровождающий (maintainer)**
```
Comment (lex111) [inline _articles/ru/leadership-and-governance.md:32]: Вы уверены, что нужно переводить термины, которые обычно транслитерируют? Почему бы не использовать, например, "мейнтейнер", если в русскоязычном сообществе в основном так принято? Как минимум, сразу становится понятным о чём речь. То же самое относится к другим словами, например вижу, что commit переводится как правки, хотя в переводе это выглядит довольно специфично, и заставляет ещё писать в скобках оригинальное название commit, так не лучше ли сразу переводить как "коммит"?

Comment (Aparus) [reply _articles/ru/leadership-and-governance.md:32]: Мы уже обсуждали это с год назад. :-) 
моя позиция такова, что текст должен быть понятен любому человеку, не только тому кто активно юзает гитхаб каждый день, но и любому, кому стала интересна тема "открытого исходника" будь то написание книги силами сообщества или другая творческая задача. 
в скобках оставляю, чтобы в случае чего было легко найти все вхождения терминологии и переправить её.

Comment (lex111) [reply _articles/ru/leadership-and-governance.md:32]: Тут дело не в гитхабе вообще, это общепринятые понятия, как раз то, что сейчас сделано только усложняет понимание, потому что зачем-то человеку нужно держать в голове перевод (иногда очень странный) и оригинальное слово, с которым он будет в основном будет работать на практике.
Например, перевод Committer как "правщик" это ведь такой же жаргон, по аналоги с контенщиком, но только ещё более специфичный, который даже не отражает реальную суть, поскольку коммитер - это всего лишь автор коммита, а тут создается впечатление, что это автор именно правки, т.е. конкретно изменения чего-либо, а не добавления, например. 
Даже в [известной и старой книге по гиту](https://git-scm.com/book/ru/v2/%D0%9E%D1%81%D0%BD%D0%BE%D0%B2%D1%8B-Git-%D0%9F%D1%80%D0%BE%D1%81%D0%BC%D0%BE%D1%82%D1%80-%D0%B8%D1%81%D1%82%D0%BE%D1%80%D0%B8%D0%B8-%D0%BA%D0%BE%D0%BC%D0%BC%D0%B8%D1%82%D0%BE%D0%B2), используется "коммитер". И по этой причине неадекватно придумывать собственное слово, которое будет только вносить путаницу с уже той терминологией, которая используется в общении.

> моя позиция такова, что текст должен быть понятен любому человеку

Это странно, каждый материал предполагает определенные предварительные знания. Например, если я хочу что-то узнать из квантовой механики, наверное мне перед этим стоит ознакомиться с основными понятиями? Здесь аналогично, только это айтишная тема, которая также предполагает начальные знания. Но главное, что англицизмы в данном случае никак не влияют на понимание материала, скорее наоборот, потому что, как минимум, человек сможет легко нагуглить термин в случае чего. Например, [пулреквест](https://htmlacademy.ru/blog/boost/tools/git-and-github-glossary) гуглится, а вот результаты по "запросу на изменение" очень разные.  И опять же, почему именно на запрос на _изменение_, может только добавить хочу? Поэтому транслитерированный вариант "пулреквест" снимает все эти вопросы. Я просто не вижу причин, почему такой вариант будет как раз менее понятен, а не наоборот.

Сравнить с другими языками нет смысла, они могли использовать тот же механизм с транслитерацией (по крайней, во французском я вижу подобное). Да даже если не было, наша задача как переводчиков, чтобы текст соответствовал стилю, правилам, которые чаще всего используются в профессиональном сообществе, это включает использование профессионального того же жаргона, сленга. А вот несколько раз указывать исходный вариант уже встречавшегося слова -- скорее антипаттерн, засоряющий текст, разумно при первом упоминании  это делать. А лучше просто использовать общепринятую терминологию, пока я не в вижу особых минусов этому.

Comment (Aparus) [reply _articles/ru/leadership-and-governance.md:32]: ну я как понял из статьи что такое опенсорс и зачем, это не айтишная тема как раз, а общая :-) 
я вообще не понимаю, зачем айтишнику читать этот перевод вместо оригинальных материалов. лично я бы хотел это закинуть знакомым гуманитариям, чтобы суть процесса мирового оценили, философию совместной работы, создания сообщества. они не захотят вникать в терминологию. 
правщик да, на ходу придумал по смыслу там это тот, у кого есть доступ на коммиты. Слова коммитер даже в англ. языке нет,  по крайней мере в [merriam-webster](https://www.merriam-webster.com/dictionary/committer) тут 
в любом случае это черновик. и любой  может править. если у вас большой опыт в этом, вы можете пройтись по всем текстам и исправить. может я сам сделаю попозже

Comment (lex111) [reply _articles/ru/leadership-and-governance.md:32]: Например, новичкам, которые входят в профессию, понадобиться этот материал, поэтому им крайне знать терминологию, и хорошо бы если она будет такая же, которая активно используется в сообществе -- им это облегчит жизнь, а у тех же специалистов не будет причин для негодования, почему переводчики в очередной раз что-то по-своему перевели, а не взяли уже общепринятый вариант.

Нужно отталкиваться от того, кому кому _в большинстве своём_ предназначена данная информация, а это, как правило, люди из IT. Вот хочу я поучаствовать в опенсорсе, читаю на своем родном языке, как там и что устроено, и когда буду готов принять участие, я уже знаю основные понятия, кто такой мейнтейнер, ведь я про него недавно читал.
Вы уже и так проделали колоссальную работу, это круто, что, наконец, это руководство будет на русском, но давайте исправим этот момент. Я могу всё сделать сам в ближайшие дни, если вы не против.

Comment (Aparus) [reply _articles/ru/leadership-and-governance.md:32]: я не против. я большую часть через гугл перевел и немного правил. всего пару статей перефразировал чтоб меньше текста было :-)

Comment (lex111) [reply _articles/ru/leadership-and-governance.md:32]: Ок, тогда заодно вычитку небольшую сделаю. Тогда, раз мы договорились, нет смысла блокировать этот пулреквест, если его не так быстро примут, отправлю правки в этот пулреквест, если нет, то открою новый.

File: _articles/ru/best-practices.md [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
-  bring-in-the-robots: "Bring in the robots"
-  its-okay-to-hit-pause: "It’s okay to hit pause"
+  what-does-it-mean-to-be-a-maintainer: 'What does it mean to be a maintainer?'
+  documenting-your-processes: 'Documenting your processes'
+  learning-to-say-no: 'Learning to say no'
+  leverage-your-community: 'Leverage your community'
+  bring-in-the-robots: 'Bring in the robots'
+  its-okay-to-hit-pause: 'It’s okay to hit pause'
 order: 5
 image: /assets/images/cards/best-practices.png
```
Comment (GDimaG) [inline _articles/ru/best-practices.md:14]: Я

Comment (Aparus) [reply _articles/ru/best-practices.md:14]: что это? почему английский текст везде?

**Comment by nholden** [comment]
👋  Hi, @Aparus! Thanks for all your work on this. It looks like [CI is still failing](https://github.com/github/opensource.guide/pull/2513/checks?check_run_id=3585673234). Could you take a look and address those issues?

**Comment by Aparus** [comment]
>  Hi, @Aparus! Thanks for all your work on this. It looks like [CI is still failing](https://github.com/github/opensource.guide/pull/2513/checks?check_run_id=3585673234). Could you take a look and address those issues?

thank you. is it because of "Warnings" ? my prettier automatically changed all list item bullets `*` to `-`. They were `warnings` and I thought that it is ok :smile:

**Comment by nholden** [comment]
> is it because of "Warnings" ?

Yes, it does look like it's because of the warnings. We should probably change that "warning" language in a future PR, because you're right that they don't _seem_ mandatory. 😅

**Comment by nholden** [comment]
Thanks for resolving those CI issues, @Aparus!

Could you talk a little bit about why the changes to `_layouts/article.html`, `package-lock.json`, and `node_modules/.package-lock.json`? Is there a way we could merge this PR with just translations?

**Comment by Aparus** [comment]
> Thanks for resolving those CI issues, @Aparus!
> 
> Could you talk a little bit about why the changes to `_layouts/article.html`, `package-lock.json`, and `node_modules/.package-lock.json`? Is there a way we could merge this PR with just translations?

Yes, you can merge just translation. 
I'm sorry about both of `package-lock`, I don't know how it happened. 
### TOC (table of content)
The change in `_layouts/article.html` -- for having TOC  in Russian version too. It's a little strange, that you switch off TOC for any languages except English. I looked at other languages, it seems like people didn't understand how to translate hashes (part of link after `#` for navigating through article headers) for TOC. Some contributors removed the TOC section completely, others kept English hashes -- and they don't work, of course. Maybe I should write instruction about how to translate hashes for TOC?

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Thanks so much for your first contribution! Without people like you submitting PRs we couldn't run this project. You rock, @Aparus!

**Comment by MikeMcQuaid** [comment]
> The change in `_layouts/article.html` -- for having TOC in Russian version too. It's a little strange, that you switch off TOC for any languages except English. I looked at other languages, it seems like people didn't understand how to translate hashes (part of link after `#` for navigating through article headers) for TOC. Some contributors removed the TOC section completely, others kept English hashes -- and they don't work, of course. Maybe I should write instruction about how to translate hashes for TOC?

@Aparus If there's any other languages that can be added here: we'd love a PR to add them. Otherwise, an addition to https://github.com/github/opensource.guide/blob/main/docs/translations.md would also make a great PR.

**Pull request merged**